c++.html.markdown

```diff
@@ -955,6 +955,53 @@ v.push_back(Foo());  // New value is copied into the first Foo we inserted
 // explanation of why this works.
 v.swap(vector<Foo>());
 
+
+///////////////////////////////////////
+// Tuples (C++11 and above)
+///////////////////////////////////////
+
+#include<tuple>
+
+// Conceptually, Tuples are similar to  old data structures (C-like structs) but instead of having named data members , 
+// its elements are accessed by their order in the tuple.
+
+// We start with constructing a tuple.
+// Packing values into tuple
+auto first = make_tuple(10,'A') ;
```

Comment:
still spaces before the `;`.

Please, look at the rest of the file, and follow the style used :).